charts/vaultwarden/templates/_podSpec.tpl

```diff
@@ -118,17 +118,25 @@ containers:
       - containerPort: 8080
         name: http
         protocol: TCP
-    {{- if or (.Values.data) (.Values.attachments) }}
+    {{- if .Values.storage.existingVolumeClaim }}
     volumeMounts:
-      {{- with .Values.data }}
+      - name: vaultwarden-data
+        mountPath: {{ default "/data" .path }}
+      - name: vaultwarden-data
+        mountPath: {{ default "/data/attachments" .path }}
```

Comment:
`.path` here is not valid anymore since this was coming from `.Values.attachments.path`.

If you still want to be able to specify a custom mount path when using an existing claim, I suggest splitting `existingVolumeClaim` into a dictionary with something like `claimName`, `dataPath`, and `attachmentsPath`. Otherwise, you can just put the path name here.

Reply:
added